app/controllers/contracts_controller.rb

```diff
@@ -1,4 +1,16 @@
 class ContractsController < ApplicationController
+
+  def index
+    if current_customer && current_customer.contracts.any?
+      @contracts = current_customer.contracts
+    elsif current_company && current_company.contracts.any?
+      @contracts = current_company.contracts
+      render 'index.html.erb'
```

Comment:
you can just omit this render because it happens automatically because of the restful naming conventions and rails default behavior. It doesn't however make sense to not put it in the if clause but then not in the elsif clause. Not preventing merge though.